packages/router-utils/tests/stripTypeExports.test.ts

```
@@ -0,0 +1,226 @@
+import { describe, expect, test } from 'vitest'
+import { parseAst, generateFromAst, stripTypeExports } from '../src/ast'
```

ESLint

[error] Member 'generateFromAst' of the import declaration should be sorted alphabetically.

(sort-imports)